src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-wed calendar--month-length-30">
```

Comment:
updates. since `wed` and `30` are modifiers
```suggestion
    <div class="calendar calendar--start-day--wed calendar--month-length--30">
```